### PR TITLE
Feature/65 add map to the search page

### PR DESCRIPTION
**Pull request by jimibilibob**

### CODE REVIEW

File: RoutesApp_ios/modules/home/controllers/HomeViewController.swift [hunk 1]
```diff
@@ -6,16 +6,122 @@
...
+
+    @IBAction func currentLocationAction(_ sender: Any) {
+        guard CLLocationManager.locationServicesEnabled() else {
+            locationManager.requestWhenInUseAuthorization()
+            return
+        }
+        // check the permission status
+        switch locationManager.authorizationStatus {
+        case .authorizedAlways, .authorizedWhenInUse:
+            print("Authorize.")
```
Comment (YawarValeriano) [inline RoutesApp_ios/modules/home/controllers/HomeViewController.swift]: Avoid using prints

File: RoutesApp_ios/modules/home/controllers/HomeViewController.swift [hunk 2]
```diff
@@ -6,16 +6,122 @@
...
+}
+
+extension HomeViewController: CLLocationManagerDelegate {
+
+    func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+        self.viewmodel.currentPosition = locationManager.location?.coordinate
+    }
+
+    func locationManager(_ manager: CLLocationManager, didFailWithError error: Error) {
+        print("error:: \(error.localizedDescription)")
```
Comment (YawarValeriano) [inline RoutesApp_ios/modules/home/controllers/HomeViewController.swift]: same here about print

**Review by YawarValeriano — CHANGES_REQUESTED** [review]

File: RoutesApp_ios/modules/home/controllers/HomeViewController.swift (hunk 2 above)
Comment (jimibilibob) [reply RoutesApp_ios/modules/home/controllers/HomeViewController.swift]: Removed

File: RoutesApp_ios/modules/home/controllers/HomeViewController.swift (hunk 1 above)
Comment (jimibilibob) [reply RoutesApp_ios/modules/home/controllers/HomeViewController.swift]: All prints removed

**Review by YawarValeriano — APPROVED** [review]

**Review by jmacboy — APPROVED** [review]

**Pull request merged**